src/status_im2/subs/communities.cljs

```diff
@@ -105,6 +105,28 @@
  (fn [communities]
    (map :id communities)))
 
+
+;; Return communities splitted by level of user participation. Some communities user
```

Comment:
Can this comment not go as a doc string in the function?
e.g
```
(re-frame/reg-sub
"doc string goes here"
...
```


also imo we only really need the last line
`Result map has form: {:joined [id1, id2] :pending [id3, id5] :opened [id4]}"`

Reply:
Unfortunately, docstring doesn't work here, because it is a function call, and looks like reg-sub doesn't support docstring (I tried).
Also, I wasn't able to remove newlines between the description and code - linter argues.

As for description, I would say that mentioning that opened communities remain in database can add a piece of information for the reader

Reply:
@vkjr, you're right about the docstring, we can't use it here, but could you join the comments with the `reg-sub` call? There are two empty lines between them.

Reply:
@ilmotta, for some reason linter returns an error when I put the comments right before the code :-/

Reply:
Such a weird recommendation from `cljfmt`! You're totally right, I guess if you prefer you can always add the comments at the top of the anonymous function, but not much of an improvement :man_shrugging: 

```clojure
(re-frame/reg-sub
 :communities/community-ids-by-user-involvement
 :<- [:communities/communities]
 ;; Return communities splitted by level of user participation. Some communities
 ;; user already joined, to some of them join request sent and others were opened
 ;; one day and their data remained in app-db. Result map has form:
 ;; {:joined [id1, id2] :pending [id3, id5] :opened [id4]}"
 (fn [communities]
   (reduce (fn [{:keys [joined pending open] :as acc} community]
             (let [joined? (:joined community)
                   requested? (pos? (:requested-to-join-at community))
                   id (:id community)]
               (cond
                 joined?    (assoc acc :joined (conj joined id))
                 requested? (assoc acc :pending (conj pending id))
                 :else      (assoc acc :opened (conj open id)))))
           {:joined [] :pending [] :opened []}
           communities)))
```

Reply:
Hmm, nice idea! For me, it really seems like the best option! I would go that way)